cypress/e2e/checkin/check-in-claimed-returned-items.cy.js

```diff
@@ -0,0 +1,199 @@
+              },
+            ],
+          }).then((specialInstanceIds) => {
+            itemsData.itemsWithSeparateInstance[index].instanceId = specialInstanceIds.instanceId;
+            itemsData.itemsWithSeparateInstance[index].holdingId = specialInstanceIds.holdingIds[0].id;
+            itemsData.itemsWithSeparateInstance[index].itemId = specialInstanceIds.holdingIds[0].itemIds;
+            itemsData.itemsWithSeparateInstance[index].materialType = testData.materialType;
+          });
+        });
+        cy.wrap(itemsData.itemsWithSeparateInstance).as('items');
+      });
+
+    PatronGroups.createViaApi(patronGroup.name).then((patronGroupResponse) => {
+      patronGroup.id = patronGroupResponse;
+    });
+
+    cy.createTempUser(
+      [
+        permissions.checkinAll.gui,
+        permissions.loansAll.gui,
+        permissions.uiUsersLoansClaimReturned.gui,
+        permissions.uiUsersView.gui,
+      ],
+      patronGroup.name
+    )
+      .then((userProperties) => {
+        userData.username = userProperties.username;
+        userData.password = userProperties.password;
+        userData.userId = userProperties.userId;
+        userData.barcode = userProperties.barcode;
+      })
+      .then(() => {
+        UserEdit.addServicePointViaApi(testData.servicePointS.id, userData.userId, testData.servicePointS.id);
+
+        cy.get('@items').each((item) => {
+          Checkout.checkoutItemViaApi({
+            id: uuid(),
+            itemBarcode: item.barcode,
+            loanDate: moment.utc().format(),
+            servicePointId: testData.servicePointS.id,
+            userBarcode: userData.barcode,
+          });
+        });
+
+        UserLoans.getUserLoansIdViaApi(userData.userId).then((userLoans) => {
+          userLoans.loans.forEach(({ id }) => {
+            UserLoans.claimItemReturnedViaApi({ itemClaimedReturnedDateTime: moment.utc().format() }, id);
+          });
+        });
+
+        cy.login(userData.username, userData.password);
+      });
+  });
+
+  after('Deleting created entities', () => {
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.servicePointS.id]);
+    ServicePoints.deleteViaApi(testData.servicePointS.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+    cy.deleteLoanType(testData.loanTypeId);
+  });
+  it(
+    'C7148 Check In: item with at least one open request (vega)',
```

Comment:
Please change the title of this autotest to 'C10974 Check In: claimed returned items (vega)'

Reply:
done